### PR TITLE
cargo-deny: update to 0.13.5.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/cargo-deny/template [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
 if [ "$XBPS_TARGET_ENDIAN" = "be" ]; then
 	broken="bitvec crate unimplemented for big endian"
 fi
 
+pre_check() {
+	rm -rf tests/advisory-db/github.com-2f857891b7f43c59
+	git clone https://github.com/rustsec/advisory-db.git tests/advisory-db/github.com-2f857891b7f43c59
+	cd tests/advisory-db/github.com-2f857891b7f43c59
+	git checkout ${_adv_commit}
+}
```
Comment (tranzystorek-io) [inline srcpkgs/cargo-deny/template:28]: We should probably fetch advisory-db as a distfile

Comment (jcgruenhage) [reply srcpkgs/cargo-deny/template:28]: Doesn't work, it wants a git checkout for the tests

Comment (tranzystorek-io) [reply srcpkgs/cargo-deny/template:28]: Ah, nevermind then

File: srcpkgs/cargo-deny/template [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
 pkgname=cargo-deny
-version=0.12.2
+version=0.13.5
 revision=1
+_adv_commit="222ccf84ef7d733a6eac71f694a5e14b4d0d0289"
 build_style=cargo
 configure_args="--no-default-features"
 hostmakedepends="pkg-config"
 makedepends="openssl-devel"
+checkdepends="git"
```
Comment (classabbyamp) [inline srcpkgs/cargo-deny/template:10]: this probably clobbers chroot-git (#39577), can we use that instead? you'll just need to add `/usr/libexec/chroot-git` to PATH

Comment (jcgruenhage) [reply srcpkgs/cargo-deny/template]: This probably also applies to python3-GitPython then, right?

Comment (classabbyamp) [reply srcpkgs/cargo-deny/template]: and probably a bunch of other things too :)

Comment (jcgruenhage) [reply srcpkgs/cargo-deny/template]: yay. In that case, wouldn't it make more sense to fix it on a lower level anyway? Knowing that you have to get certain tools from certain other paths is increasing the barrier-to-entry for new maintainers quite a bit IMO

Comment (classabbyamp) [reply srcpkgs/cargo-deny/template]: yes, it does need a lower-level fix (as discussed in the issue) but we shouldn't break things in the meantime

Comment (jcgruenhage) [reply srcpkgs/cargo-deny/template]: Agreed, which is why I adjusted this to not require `git` in `checkdepends` for now. I'll mark this as resolved.

**Pull request merged**